files/build_templates/docker_image_ctl.j2

```diff
@@ -181,7 +186,13 @@ start() {
         NET="container:database$DEV"
         {%- endif %}
     fi
-
+{%- if docker_container_name == "bgp" %}
+    if [ "$DEV" ]; then
+        if [ ! -d "/etc/sonic/frr$DEV" ]; then
```

Comment:
>/etc/sonic/frr$DEV [](start = 19, length = 18)

If you use directory structure like `/etc/sonic/frr/$DEV`, we could keep code clean, and no need to change docker mount.

Reply:
Leads to a bad directory structure upon copying frr directory. will keep this as is.

Reply:
Fixed.